### PR TITLE
add missing claim targets and common core codes

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: warehouse/sql/V201702061486427095__initial_dml.sql [hunk 1]
```diff
@@ -1389,6 +1489,10 @@ INSERT INTO common_core_standard ( subject_id, natural_id, description) VALUES
 (1,'A-REI.7','Solve a simple system consisting of a linear equation and a quadratic equation in two variables algebraically and graphically. For example, find the points of intersection between the line y = –3x and the circle x^2 + y^2 = 3.'),
 (1,'A-REI.8','Represent a system of linear equations as a single matrix equation in a vector variable.'),
 (1,'A-REI.9','Find the inverse of a matrix if it exists and use it to solve systems of linear equations (using technology for matrices of dimension 3 × 3 or greater).'),
+(1,'A-REI.A','Understand solving equations as a process of reasoning and explain the reasoning.'),
```
Comment (malaffoon) [inline warehouse/sql/V201702061486427095__initial_dml.sql]: For the common-core codes there is a site that defines all of them, so i got the descriptions from there. The existing descriptions seem to match those from that site so i'm pretty sure it's okay.

File: warehouse/sql/V201702061486427095__initial_dml.sql [hunk 2]
```diff
@@ -1405,22 +1509,29 @@ INSERT INTO common_core_standard ( subject_id, natural_id, description) VALUES
...
 (1,'8.EE.3','Use numbers expressed in the form of a single digit times an integer power of 10 to estimate very large or very small quantities, and to express how many times as much one is than the other. For example, estimate the population of the United States as 3 × 10^8 and the population of the world as 7 × 10^9, and determine that the world population is more than 20 times larger.'),
 (1,'8.EE.4','Perform operations with numbers expressed in scientific notation, including problems where both decimal and scientific notation are used. Use scientific notation and choose units of appropriate size for measurements of very large or very small quantities (e.g., use millimeters per year for seafloor spreading). Interpret scientific notation that has been generated by technology.'),
-(1,'7.EE.1','Apply properties of operations as strategies to add, subtract, factor, and expand linear expressions with rational coefficients.'),
```
Comment (malaffoon) [inline warehouse/sql/V201702061486427095__initial_dml.sql]: I didn't delete any entries, just moved around a few to better order them.

File: warehouse/sql/V201702061486427095__initial_dml.sql [hunk 3]
```diff
@@ -225,6 +225,7 @@ INSERT INTO target (claim_id, natural_id, code, description) VALUES
 ((select id from claim where subject_id = 1 and code = '1'),'G-GPE|X','X','Geometry: Translate between the geometric description and the equation for a conic section.'),
 ((select id from claim where subject_id = 1 and code = '1'),'G-GPE|XX','XX','Geometry: Use coordinates to prove simple geometric theorems algebraically.'),
 ((select id from claim where subject_id = 1 and code = '1'),'G-MG|X','X','Geometry: Apply geometric concepts in modeling situations.'),
+((select id from claim where subject_id = 1 and code = '1'),'G-SRT|O','P','UNKNOWN'),
 ((select id from claim where subject_id = 1 and code = '1'),'G-SRT|P','P','Geometry: Define trigonometric ratios and solve problems involving right triangles.'),
```
Comment (agorina) [inline warehouse/sql/V201702061486427095__initial_dml.sql]: Should probably add a TODO or create a JIRA to clean up the 'UNKNOWN' before the release.

Comment (malaffoon) [inline warehouse/sql/V201702061486427095__initial_dml.sql]: https://jira.fairwaytech.com/browse/DWR-417

**Pull request merged**